control-plane/catalog/registration/cache.go

```diff
@@ -239,13 +332,17 @@ func (c *RegistrationCache) removeTermGWACLRole(log logr.Logger, registration *v
 		return err
 	}
 
-	roles, _, err := client.ACL().RoleList(nil)
-	if err != nil {
-		return err
-	}
-
 	var mErr error
 	for _, termGW := range termGWsToUpdate {
+
+		queryOpts := &capi.QueryOptions{}
+
+		writeOpts := &capi.WriteOptions{}
```

Comment:
~~What do we get by defining these as empty structs instead of continuing to pass `nil` below?~~

See above comment